src/tiles/grid/item-tile.ts

```diff
@@ -1,23 +1,323 @@
-import { css, html, LitElement } from 'lit';
+import { css, CSSResultGroup, html, LitElement } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 import { TileModel } from '../../models';
+import { formatCount } from '../../utils/format-count';
+
+import '../../mediatype-icon';
+
+import { favoriteFilledIcon } from './icons/favorite-filled';
+import { reviewsIcon } from './icons/reviews';
+import viewsIcon from './icons/views';
 
 @customElement('item-tile')
-export class ItemTIle extends LitElement {
+export class ItemTile extends LitElement {
   @property({ type: Object }) model?: TileModel;
 
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  get renderItemImageView() {
+    const imgSrcUrl = `${this.baseNavigationUrl}/services/img/${this.model?.identifier}`;
+    const containsDeemphasize = this.model?.collections.includes('deemphasize');
+    return !containsDeemphasize
+      ? html`<div
+          id="item-image"
+          style="background-image:url(${imgSrcUrl})"
+        ></div>`
+      : html`<div id="item-image-box">
+          <div
+            id="item-image-deemphasize"
+            style="background-image:url(${imgSrcUrl})"
+          ></div>
+          <div class="tile-action no-preview">Content may be inappropriate</div>
+        </div>`;
+  }
+
   render() {
+    const collectionName = this.model?.collections[0];
+    const collectionUrl = `${this.baseNavigationUrl}/details/${collectionName}`;
+
+    const imgSrcUrl = `${this.baseNavigationUrl}/services/img/${this.model?.collections[0]}`;
+    const itemTitle = this.model?.title || '';
+    const itemCreator = this.model?.creator || '-';
+
     return html`
-      <h1>${this.model?.title}</h1>
-      <h2>${this.model?.datePublished?.toDateString()}</h2>
-      <p>${this.model?.description?.substring(0, 100)}...</p>
+      <div id="container">
+        <a href=${collectionUrl}>
+          <div id="stealth-popup">
+            <div id="collection-thumbnail" style="background-image:url(${imgSrcUrl})"></div>
+            <div id="collection-title-text">${collectionName}</div>
+          </div>
+        </a>
+        <div id="title-image-container">
+          <h1 id="item-title" title=${itemTitle}>
+            ${this.model?.title}
+          </h1>
+          <div id="item-image-container">
+            ${this.renderItemImageView}
+          </div>
+          <div class="item-creator">
+            <span id="text-by">By:</span>
+            <span>${itemCreator}</span>
+          </div>
+        </div>
+        <div id="item-stats-container">
+          <div id="stats-holder">
+            <div class="col">
+              <mediatype-icon mediatype="${
+                this.model?.mediatype
+              }" showText="true">
```

Comment:
In lit, when you're binding values from a property like this, it's typically best practice to use lit [bindings](https://lit.dev/docs/templates/expressions/) (put the dot or question mark before the attribute, depending on the type). This allows that property to be updated if the source value changes. ie:

```suggestion
              <mediatype-icon .mediatype=${
                this.model?.mediatype
              } ?showText=${true}>
```